packages/amplify-frontend-javascript/tests/frontend-config-creator.test.js

```diff
@@ -62,6 +62,18 @@ module.exports = awsmobile;
     // expect(configCreator.getCurrentAWSExports(context)).toThrowError('Unable to parse aws-exports.js. Has this file been modified?');
   });
 
+  it('should load with babel.config.json present', async () => {
```

Comment:
It will be better to include a mock test with ```.babelrc``` file  not to be read  from the client project

Reply:
I tried that but I was unable to get that case failing in the jest test. Therefore I parked that effort to unblock release.